src/hotspot/share/runtime/threads.hpp

```diff
@@ -131,7 +131,7 @@ class Threads: AllStatic {
   // Print threads busy compiling, and returns the number of printed threads.
   static unsigned print_threads_compiling(outputStream* st, char* buf, int buflen, bool short_form = false);
 
-  // Get Java threads that are waiting to enter a monitor.
+  // Get count Java threads that are waiting to enter or re-enter the specified monitor.
```

Comment:
nit typo: s/count Java/count of Java/

Also this file needs a copyright year update.

Reply:
Thanks, fixed now.